Lib/fontParts/base/segment.py

```diff
@@ -24,6 +19,19 @@ def _reprContents(self):
             contents.append("index='%r'" % self.index)
         return contents
 
+    def __eq__(self, other):
+        if hasattr(other, "points"):
+            return self.points == other.points
+        return NotImplemented
+
+    def __ne__(self, other):
+        if hasattr(other, "points"):
+            return not self.points == other.points
+        return NotImplemented
+
+    def __hash__(self):
+        return hash(self.points)
```

Comment:
I'm confused.

If the aim is to fix https://github.com/robofab-developers/fontParts/issues/52
then you DON'T want this hash function. You just want `id(self)`. Same with __eq__.

What you have now, is not good hashable type, as self.points can change, right?

Reply:
Got it, thank you!